cmd/jiralert/main.go

```diff
@@ -1,3 +1,16 @@
+// Copyright 2017 The Prometheus Authors
```

Comment:
hmmm It's not prometheus authors. I think we have to put some jiralert authors or something instead :thinking: 
cc @free

Reply:
I think it is ok since @free is part of the prometheus-community team than we are all part of the Promethues Authors.